_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -0,0 +1,535 @@
+- **different hue**: the element's [foreground colors][foreground color] and the [foreground colors][foreground color] of the **non-link line text** elements have a [different hue][], or the element's [background colors][background color] and the [background colors][background color] of the **non-link line text** elements have a [different hue][].
+
+## Expectation
+
+For all [interaction states][] there exists at least one [visible][] [inclusive descendant][] and one [adequate inspection state][] of the test target such that at least one of the following is true:
+
+- **distinguishing style**: the [visible][] [inclusive descendant][] has a [distinguishable style][] from the **non-link line text** elements; or
+- **distinguishing content**: the [visible][] [inclusive descendant][] has adjacent content (such as an image or text) that indicates the test target is a link.
+
+## Assumptions
+
+- Any change in font is sufficiently distinguishable, and fonts are loaded when they are present.
+- The same foreground color and the same background color is used by all the visible text nodes of the _ancestor_ element that are not part of the target element, otherwise color can not be a distinguishing factor.
+- The same `box-shadow` is used by all the visible text nodes of the _ancestor_ element that are not part of the target element, otherwise `box-shadow` can not be a distinguishing factor.
+- The same `border` is used by all the visible text nodes of the _ancestor_ element that are not part of the target element, otherwise `border` can not be a distinguishing factor.
+- The perceived visual styling of elements is the result of CSS styling applied to those elements and inherited from their _ancestor_ elements. Through manipulation of the positioning and styles of other elements, including non inherited styles of _ancestor_ elements, an element may end up with a perceived distinguishable style not resulting from its own and inherited styling. In that case, this rule might fail while the [Success Criterion 1.4.1 Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html) might still be satisfied.
+
+## Accessibility Support
+
+_No accessibility support issues known._
+
+## Background
+
+- [Understanding Success Criterion 1.4.1: Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html)
+- [G183: Using a contrast ratio of 3:1 with surrounding text and providing additional visual cues on focus for links or controls where color alone is used to identify them](https://www.w3.org/WAI/WCAG21/Techniques/general/G183)
+- [F73: Failure of Success Criterion 1.4.1 due to creating links that are not visually evident without color vision](https://www.w3.org/WAI/WCAG21/Techniques/failures/F73)
+- [C15: Using CSS to change the presentation of a user interface component when it receives focus](https://www.w3.org/WAI/WCAG21/Techniques/css/C15)
+
```

Comment:
I think we should add a background note in the line of "default styling is good; we need the examples to override it to illustrate the rule, but you shouldn't". To limit the risk of people seeing the Passed Examples and believing they need to do the same.

Reply:
I've added a note about this. Please review it.